README.md

```diff
@@ -1,13 +1,58 @@
 # cQASM language specification
 
-This repository contains the specification documents of the cQASM quantum programming language. 
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-_The specification hosted through GitHub Pages under the following link:_ 
+This repository contains the language specification of the cQASM quantum programming language.
+cQASM stands for *c*ommon *Q*uantum *AS*se*M*bly language.
 
-### [<img src="./docs/_static/book.svg"> cQASM language specification](https://qutech-delft.github.io/cQASM-spec/) 
+## Deployment
 
-Documentation on development and deployment can be found in the [Wiki](https://github.com/QuTech-Delft/cQASM-spec/wiki).
+### MkDocs
+
+The cQASM language specification documentation is generated using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/),
+a documentation framework on top of [MkDocs](https://www.mkdocs.org).
+
+    mkdocs.yml                   # The configuration file
+    mkdocs-base.yml              # Base configuration file
+    docs/
+        index.md                 # The documentation homepage
+        introduction/            # History, context, acknowledgements
+        language_specification/  # The cQASM language specification
+        appendices/              # Spin-2
+        javascripts/             # JS configuration files for extensions
+
+You can serve the documentation locally at <http://localhost:8000> by running:
+
+```shell
+mkdocs serve
+```
+
+The documentation is hosted through GitHub Pages. To deploy the docs run:
+
+```shell
+mkdocs gh-deploy
+```
+
+### Using the Docker container
+
+The documentation can also be viewed via the provisioned Docker container.
+Run the docker container with the following command:
+
+```shell
+docker compose up -d
+```
+
+The documentation can now be viewed at <http://localhost:8106>.
```

Comment:
Move to confluence page.

Reply:
Created: https://qutech-sd.atlassian.net/wiki/spaces/QW/pages/3555950593/Deployment.